server/src/main/java/org/elasticsearch/index/mapper/TextFieldMapper.java

```diff
@@ -980,9 +985,6 @@ public IndexFieldData.Builder fielddataBuilder(FieldDataContext fieldDataContext
                 );
             }
 
-            if (operation != FielddataOperation.SCRIPT) {
```

Comment:
why isn't this needed with the third enum option anymore? as far as I can see this so far triggered for `SEARCH`, why not `(operation == FielddataOperation.SEARCH)` then now?